code/datums/wires/simple_wire.dm

```diff
@@ -0,0 +1,22 @@
+/datum/wires/simple_wire   //two bit wire type just to add assemblies to things that dont have specific wires
+	randomize = 0
+
```

Comment:
They do not serve any purpose what so ever
All the usual wire procs are completely overridden with no backtracking thus they dont need anything really